content/api/resources/trades/list.mdx

```diff
@@ -48,6 +51,9 @@ When filtering for a specific orderbook, you must use use all six of these argum
 - counter_asset_code
   - optional
   - The code for the counter asset.
+- trade_type
+  - optional
+  - Can be set to `liquidity_pools` to filter only trades executed across a liquidity pool.
```

Comment:
It can also be set to `orderbook` and `all` (default), see [here](https://docs.google.com/document/d/1pXL8kr1a2vfYSap9T67R-g72B_WWbaE1YsLMa04OgoU/edit#heading=h.bexstdt2tlbj).